php/tests/generated_class_test.php

```diff
@@ -877,4 +877,279 @@ public function testPrefixForReservedWords()
         $m = new \Foo\PBEmpty();
         $m = new \PrefixEmpty();
     }
+
+    #########################################################
```

Comment:
I prefer to add a new test file for documentation. This is not our API and is open to change.